config/samples/service.yaml

```diff
@@ -1,32 +0,0 @@
-kind: Service
-apiVersion: v1
-metadata:
-  name:  mylbapp
-  annotations:
-    lb.kubesphere.io/v1alpha1: porter
-spec:
-  selector:
-    app:  mylbapp
-  type:  LoadBalancer 
-  ports:
-  - name:  http
-    port:  8088
-    targetPort:  80
-
----
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  name: mylbapp
-spec:
-  replicas: 3
-  template:
-    metadata:
-      labels:
-        app: mylbapp
-    spec:
-      containers:
-      - name: name
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
```

Comment:
Why was this file removed?

Reply:
This is supposed to be just an example to explain how to use it, but if you want to know how to use it, you can refer to the documentation at the bottom of the doc directory.

Reply:
The README refers to this file which (on master) doesn't exist anymore.

Reply:
ok， i see,  thanks for the reminder